templates/terraform/examples/container_analysis_occurrence_kms.tf.erb

```diff
@@ -0,0 +1,51 @@
+resource "google_binary_authorization_attestor" "attestor" {
+  name = "<%= ctx[:vars]["attestor"] %>"
+  attestation_authority_note {
+    note_reference = google_container_analysis_note.note.name
+    public_keys {
```

Comment:
things to consider as follow up - this might need to be a set? I didn't test it in-depth in this PR but I'd probably split it out anyways